crates/oxc_transformer/src/regexp/mod.rs

```diff
@@ -242,10 +242,12 @@ fn try_parse_pattern<'a>(
     flags: RegExpFlags,
     ctx: &mut TraverseCtx<'a>,
 ) -> Result<Pattern<'a>> {
-    use oxc_regular_expression::{Parser, ParserOptions};
-
-    let options = ParserOptions::default()
-        .with_span_offset(span.start + 1) // exclude `/`
-        .with_flags(&flags.to_string());
-    Parser::new(ctx.ast.allocator, raw, options).parse()
+    use oxc_regular_expression::{LiteralParser, Options};
+
+    let _flags_text = &flags.to_string(); // TODO: How to add `&'a str` lifetime?
```

Comment:
Now I am having trouble to resolve this lifetime constraint due to my lack of Rust experience...